### ISSUE TITLE
Where clause not working with graphql: "15.0.0"

### ISSUE THREAD

**Issue by zacksayenko** [join-monster/join-monster#405]
I was just setting up a new project and noticed that using "graphql": "^15.0.0" didn't work well with the latest join-monster package.

Main issue that I noticed is that where clause wasn't honored.

Here are the packages that I was using:
```
  "dependencies": {
    "express": "^4.17.1",
    "express-graphql": "^0.9.0",
    "graphql": "^15.0.0",
    "join-monster": "^2.1.0",
    "knex": "^0.21.1",
    "pg": "^8.0.3"
  }
```

Relevant parts of the code:

```
const Account = new graphql.GraphQLObjectType({
    name: 'Account',
    fields: () => ({
      id: { type: graphql.GraphQLString },
      name: { type: graphql.GraphQLString },
      email: { type: graphql.GraphQLString },
    }),
  });
  
  Account._typeConfig = {
    sqlTable: 'account',
    uniqueKey: 'id',
  }

const QueryRoot = new graphql.GraphQLObjectType({
    name: 'Query',
    fields: () => ({
      hello: {
        type: graphql.GraphQLString,
        resolve: () => "Health check successful!"
      },
      account: {
        type: Account,
        args: { 
            id: { type: graphql.GraphQLNonNull(graphql.GraphQLString) } 
        },
        where: (accountTable, args, context) => {
            return `${accountTable}.id = '${args.id}'`
        },
        resolve: (parent, args, context, resolveInfo) => {
          return joinMonster.default(resolveInfo, {}, sql => {
            console.log(sql);
            return knex.raw(sql);
          });
        }
      },
    })
  });
  
  const connectedSchema = new graphql.GraphQLSchema({
    query: QueryRoot,
  });
```
 
When using graphql 15, the sql statement that was produced inside the resolver didn't have the where clause.
As soon as I downgraded to 14.1.1 all worked fine.

**Comment by Auronmatrix** [comment]
+1. Can confirm this. Where clause is ignored in graphql 15.0.0

**Comment by JCMais** [comment]
This library needs to be changed to use the `extensions` object instead of adding new fields directly to the GraphQL definitions.

The extensions field was added here https://github.com/graphql/graphql-js/pull/2097

**Comment by alexbbt** [comment]
This is already being worked on here: https://github.com/join-monster/join-monster/pull/418

**Comment by lorensr** [comment]
Published `3.0.0-alpha.1`. @zacksayenko let me know if it works for you with graphql@15! Note the new `where` format: https://join-monster.readthedocs.io/en/latest/where/

**Comment by mnunezdm** [comment]
when is this version going to be released? ive been using it since august with no issues

**Comment by lorensr** [comment]
Where clause should work in the v3 beta. Let us know if you have problems with it!

**Comment by bwind00** [comment]
How do I get v3 beta? We have an urgent need to get this working, join-monster with the latest graphql. Thanks!

**Comment by lorensr** [comment]
npm i join-monster@alpha

**Comment by bwind00** [comment]
thanks, but that just installed 3.0.0-alpha2 which has the same problem with where mentioned above

**Comment by bwind00** [comment]
Sorry, an ingenious colleague figured out what we were doing wrong. In case it helps others, we had forgotten to encapsulate the where attribute in the new extensions attribute...
      **extensions: {**
          joinMonster: {
            where: (att, args, context) => {
             return `${att}."ATRBT_ID" = :atrbtId`
            }
          }
      **}**

**Comment by lorensr** [comment]
Glad to hear! And just released as v3.0.0
